daffodil-japi/src/main/scala/org/apache/daffodil/japi/Daffodil.scala

```diff
@@ -490,10 +490,15 @@ class DataLocation private[japi] (dl: SDataLocation) {
   override def toString() = dl.toString
 
   /**
-   * Determine if this data location is at the end of the input data
+   * Determine if this data location is known to be at the end of the data stream.
```

Comment:
Revert comment. isAtEnd has the original blocking behavior. It's just deprecated now because that functionality is so problematic.